src/doc/users-guide/install.md

```diff
@@ -99,42 +99,54 @@ the `--help` argument to the script.
 
 The OpenCilk C (or C++) compiler can be invoked via `bin/clang` (or
 `bin/clang++`) from within the installation directory.
+Alternatively, you can configure your system so that `clang` and `clang++` point to the OpenCilk C and C++ compilers, 
+as shown in the example below.
 
-{% alert "info" %}
-***Example:***
+### Example
+
+{% alert "primary" %}
 
-The following shows the process of installing OpenCilk into a version-specific
-subdirectory within `/opt/opencilk/`.
+The following example shows the process on Ubuntu 20.04 to install OpenCilk into `/opt/opencilk/`
+without adding a version-specific subdirectory.
+The installation and setup process is analogous for macOS and other Linux systems.
+
+- Download the precompiled [OpenCilk shell
+archive](/doc/users-guide/install/#installing-using-a-shell-archive) for your
+system.  
+- Make directory `/opt/opencilk` if it does not already exist, and execute the shell script to install OpenCilk into the directory.
```

Comment:
There's some inconsistency  in this example with using either `/opt/opencilk` (with no final slash) or `/opt/opencilk/` (with a final slash).  It looks like some of that inconsistency was there before, but the page should probably be consistent.